pyproject.toml

```diff
@@ -69,3 +69,20 @@ norecursedirs = [
     [[tool.towncrier.section]]
         name = ""
         path = ""
+
+
+[tool.ruff]
+target-version = "py39"
+
+[tool.ruff.lint]
+select = ["E", "F", "I"]
```

Comment:
I suggest adding a few here (perhaps in a separate PR if they are too strict):
- `NPY` (numpy rules)
- `PTH` (flake8-use-pathlib)
-  `UP` (pyupgrade)

too much for now, but maybe in the future...:
- `PERF` (performance checks) - currently finds 12 suggestions
-  `SIM` (simplify code)
-  `T20` (print statement), but exclude examples and the ctapipe-info tool
-  `D` (pydocstyle) - we have a  lot of issues here.

Reply:
Yes, I planned to add iteratively. Pydocstyle as well